common/src/lib.rs

```diff
@@ -114,41 +97,31 @@ pub struct Login {
 impl ClientMessageType for Login {}
 
 #[derive(Debug, Clone, Serialize, Deserialize)]
-pub enum ServerMessage {
-    Response {
-        response: RequestResponse,
-        request_id: RequestId,
-    },
+pub enum ClientboundPayload {
```

Comment:
The first uses "Message" and not "Payload"

Reply:
The first?

Reply:
"ServerboundMessage"

Reply:
should we change that? or how could we name these ones differently?

Reply:
They should both use Message or Payload - personally, I prefer Message.

Reply:
There's `ClientboundMessage` and `ClientboundPayload`